### PR TITLE
fix(wippy): EE2-1983 fix wippy replace on windows

### PR DESCRIPTION
**Pull request by igordolgovspiralscout**
### Summary
Fixes module path resolution in `wippy replace` to work correctly on Windows. Refactors path resolution logic for cross-platform compatibility.

#### Core Fixes
- **`createProjectRootFS`**: Uses `filepath.Abs()` and `filepath.Clean()` instead of `os.Getwd()` to properly handle Windows paths
- **`resolveModulePath`**: Complete rewrite using `filepath.IsAbs()` and `filepath.Rel()` for cross-platform path handling
- **Base path resolution**: Uses lock file directory instead of current working directory as the base for resolving module paths
- **Module loading**: Handles modules outside project root by creating separate filesystems

#### Testing
  - Absolute and relative paths
  - Cross-platform path separators
  - Windows-specific scenarios
  - Edge cases (empty paths, multiple slashes, special characters)
- Fixed flaky test in `system/eventbus/bus_test.go`

### CODE REVIEW

File: cmd/runner/app/app.go [hunk 1]
```diff
@@ -901,28 +901,38 @@ func createProjectRootFS(_ string) (iofs.FS, error) {
...
-				zap.String("finalModulePath", resolvedPath))
+		// Convert absolute path to relative path from current working directory
+		relPath, err := filepath.Rel(currentDir, modulePath)
+		if err != nil {
+			return "", fmt.Errorf("failed to resolve relative path from %s to %s: %w", currentDir, modulePath, err)
+		}
 
-			return resolvedPath, nil
+		// Check if the path escapes from the current directory (security check)
+		if strings.HasPrefix(relPath, "..") {
```
Comment (rustatian) [inline cmd/runner/app/app.go]: I'm not sure about this check. Usually, this security check is used for HTTP (or external) requests, when you use the PATH from it. Unverified sources. But here, if somebody has access to overwrite a lock or module path directly on a server (internal file system)... You have a much bigger problem than checking for `..` into `/etc/passwd`. So I think this is a pointless check.

Comment (igordolgovspiralscout) [reply cmd/runner/app/app.go]: removed defense for path traversal attacks

**Review by wolfy-j — APPROVED** [review]

**Review by rustatian — APPROVED** [review]

**Comment by wolfy-j** [comment]
This code is outdated, in se branch I have new implementation of lock file with proper composition, can you please check if this issue still exists there.
